src/function/FunctionElem.ts

```diff
@@ -0,0 +1,71 @@
+import {graph} from "../graph/Graph";
+import {drawGraphElement, highlightCell, unHighlightCell} from "./FunctionDraw";
+import {getElementShape} from "./FunctionGetVars";
+import {paper} from "../main/DiagramCanvas";
+import {PackageNode} from "../datatypes/PackageNode";
+import {graphElement} from "../graph/GraphElement";
+import {addClass, addVocabularyElement, createNewElemIRI} from "./FunctionCreateVars";
+import {parsePrefix} from "./FunctionEditVars";
+import {ProjectElements, ProjectSettings, VocabularyElements} from "../config/Variables";
+
+export function resizeElem(id: string) {
+	let view = paper.findViewByModel(id);
+	if (view) {
+		let bbox = view.getBBox();
+		let cell = graph.getCell(id);
+		let links = graph.getConnectedLinks(cell);
+		for (let link of links) {
+			if (link.getSourceCell()?.id === id) {
+				link.source({x: bbox.x, y: bbox.y});
+			} else {
+				link.target({x: bbox.x, y: bbox.y});
+			}
+		}
+		if (typeof cell.id === "string") {
+			unHighlightCell(cell.id)
+			highlightCell(cell.id);
+		}
+		for (let link of links) {
+			if (link.getSourceCell() === null) {
+				link.source({id: id, connectionPoint: {name: 'boundary', args: {selector: getElementShape(id)}}});
+			} else {
+				link.target({id: id, connectionPoint: {name: 'boundary', args: {selector: getElementShape(id)}}});
+			}
+		}
+	}
+}
+
+export function createNewConcept(point: { x: number, y: number }, name: { [key: string]: string }, language: string, pkg: PackageNode) {
```

Comment:
you are using this type quite heavily, maybe an explicitely defined type would help